packages/react-native-editor/jest_ui_test_environment.js

```diff
@@ -18,8 +15,7 @@ class CustomEnvironment extends JSDOMEnvironment {
 	async setup() {
 		try {
 			await super.setup();
-			this.global.editorPage = await initializeEditorPage();
-			this.global.editorPage.blockNames = blockNames;
```

Comment:
`blockNames` are usually imported from `editor-page` module, so we don't need to export it globally.